depscan/cli.py

```diff
@@ -744,6 +744,22 @@ def main():
         pkg_vulnerabilities,
         pkg_group_rows,
     ) = (None, None, None, None, None, None)
+    if os.getenv("GITHUB_ACTIONS").lower() == "__appthreat_dep-scan-action" \
```

Comment:
Use `GITHUB_ACTION` without the s.
GITHUB_ACTION	The name of the action currently running, or the [id](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsid) of a step. For example, for an action, __repo-owner_name-of-action-repo.

As per the docs, this value will be set to true.

GITHUB_ACTIONS	Always set to true when GitHub Actions is running the workflow. You can use this variable to differentiate when tests are being run locally or by GitHub Actions.

Reply:
@prabhu Good catch.